paddle/fluid/operators/distributed/communicator.cc

```diff
@@ -15,6 +15,7 @@ limitations under the License. */
 #include "paddle/fluid/operators/distributed/communicator.h"
```

Comment:
Can you move communicator into paddle/fluid/framework/fleet/? Since communicator is a singleton instance that can be called from anywhere, it is not reasonable to put communicator under paddle/fluid/operators.

Reply:
ok can do this in next pr